### PR TITLE
Remove last_completion_value.

### PR DESCRIPTION
**Pull request by zherczeg**
Code cleaning, not much effect on performance.

Benchmark | Perf (sec) |
----: | ----: | 
3d-cube.js | 1.490 -> 1.497 : -0.507% | 
3d-raytrace.js | 1.539 -> 1.551 : -0.763% | 
access-binary-trees.js | 0.763 -> 0.764 : -0.165% | 
access-fannkuch.js | 3.253 -> 3.164 : +2.724% | 
access-nbody.js | 1.589 -> 1.575 : +0.901% | 
bitops-3bit-bits-in-byte.js | 0.793 -> 0.799 : -0.744% | 
bitops-bits-in-byte.js | 1.043 -> 1.050 : -0.681% | 
bitops-bitwise-and.js | 1.945 -> 1.984 : -1.986% | 
bitops-nsieve-bits.js | 2.559 -> 2.530 : +1.154% | 
controlflow-recursive.js | 0.548 -> 0.552 : -0.563% | 
crypto-aes.js | 1.480 -> 1.465 : +0.970% | 
crypto-md5.js | 0.929 -> 0.926 : +0.311% | 
crypto-sha1.js | 0.872 -> 0.871 : +0.109% | 
date-format-tofte.js | 1.173 -> 1.177 : -0.343% | 
date-format-xparb.js | 0.612 -> 0.598 : +2.264% | 
math-cordic.js | 2.000 -> 2.007 : -0.341% | 
math-partial-sums.js | 1.124 -> 1.100 : +2.121% | 
math-spectral-norm.js | 0.888 -> 0.893 : -0.560% | 
string-base64.js | 4.736 -> 4.691 : +0.951% | 
string-fasta.js | 2.267 -> 2.225 : +1.865% | 
Geometric mean: | +0.343% |

### CODE REVIEW

File: jerry-core/ecma/base/ecma-helpers-value.c [hunk 1]
```diff
@@ -320,7 +320,7 @@ ecma_is_value_object (ecma_value_t value) /**< ecma value */
 inline bool __attr_pure___ __attr_always_inline___
 ecma_is_value_error (ecma_value_t value) /**< ecma value */
 {
-  return (value & ECMA_VALUE_ERROR_FLAG) != 0;
+  return unlikely ((value & ECMA_VALUE_ERROR_FLAG) != 0);
```
Comment (LaszloLango) [inline jerry-core/ecma/base/ecma-helpers-value.c]: This change looks unrelated. Do we really need it in this patch?

**Comment by zherczeg** [comment]
Removed

**Comment by LaszloLango** [comment]
@zherczeg, thanks. LGTM

**Comment by dbatyai** [comment]
LGTM
